tests/network/setup_multimachine.pm

```diff
@@ -39,6 +39,8 @@ sub run {
     # Set the hostname to identify both minions
     set_hostname $hostname;
 
+    # Perform ping to client from server
+    assert_script_run("ping -M do -c 1 -s 1430 10.0.2.102") if ($hostname =~ /server|master/);
```

Comment:
@DeepthiYV PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17817 was merged, you can reuse the function which was introduced there (either with multiple sizes check or with fixed size).